css/index.css

```diff
@@ -0,0 +1,70 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
This is a compiled css file did you mean to commit this?

Reply:
Nope, removing